### PR TITLE
Update our Intro page - #1 article 

### LINKED ISSUE #3496: The first preview release of ASP.NET came out almost 15 years ago as part of the .NET Framework.
From https://docs.microsoft.com/en-us/aspnet/core/
Fix/update that.

### PR DESCRIPTION
**Pull request by Rick-Anderson**
[Review URL](https://review.docs.microsoft.com/en-us/aspnet/core/?branch=pr-en-us-3497)
Given this is our #1 article we should put some effort into it. After we agree, let's run it by Dan.
Fixes  #3496

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@tdykstra  @scottaddie  for changes I'm likely to accept,  edit in browser and create a PR.

File: aspnetcore/index.md [hunk 1]
```diff
@@ -16,66 +16,47 @@ uid: index
...
+* Integration of [modern client-side frameworks](client-side/index.md) and development workflows.
+* A cloud-ready environment-based [configuration system](fundamentals/configuration.md).
+* Built-in [dependency injection](fundamentals/dependency-injection.md).
+* A light-weight and modular HTTP request pipeline.
+* Ability to host on IIS or self-host in your own process.
+* Built on [.NET Core](https://microsoft.com/net/core), which supports true side-by-side app versioning.
+* Ships entirely as [NuGet](https://nuget.org)  packages.
+* New tooling that simplifies modern web development.
+* Build and run cross-platform ASP.NET Core apps on Windows, Mac, and Linux.
 * Open source and community focused
```
Comment (tdykstra) [inline aspnetcore/index.md]: hyphenate both

File: aspnetcore/index.md [hunk 2]
```diff
@@ -16,66 +16,47 @@ uid: index
...
 
 ## Build web APIs and web UI using ASP.NET Core MVC
 
-* You can build HTTP services that reach a broad range of clients, including browsers and mobile devices.  Support for [multiple data formats and content negotiation](mvc/models/formatting.md) is built-in. ASP.NET Core is an ideal platform for building web APIs and RESTful apps on .NET Core. See [Building web APIs](tutorials/index.md#building-web-apis).  
-
+* Build HTTP services that reach a broad range of clients, including browsers and mobile devices.  Support for [multiple data formats and content negotiation](mvc/models/formatting.md) is built-in. ASP.NET Core is an ideal platform for building web APIs and RESTful apps on .NET Core. See [Building web APIs](tutorials/index.md#building-web-apis).
 * You can create well-factored and testable web apps that follow the Model-View-Controller (MVC) pattern. See [MVC](mvc/index.md) and [Testing](testing/index.md).
```
Comment (tdykstra) [inline aspnetcore/index.md]: Why remove "you can" above but not here?

File: aspnetcore/index.md [hunk 3]
```diff
@@ -16,66 +16,47 @@ uid: index
...
-For getting-started tutorials, see [ASP.NET Core Tutorials](tutorials/index.md)
-
-For in-depth introduction to ASP.NET Core concepts and architecture, see [ASP.NET Core Fundamentals](fundamentals/index.md).
+ASP.NET Core is designed to integrate seamlessly with a variety of client-side frameworks, including [AngularJS](client-side/angular.md), <!-- KO is no longer featured --> [KnockoutJS](client-side/knockout.md) and [Bootstrap](client-side/bootstrap.md). See [Client-Side Development](client-side/index.md) for more details.
 
-An ASP.NET Core app can use the .NET Core or .NET Framework runtime. For more information, see [Choosing between .NET Core and .NET Framework](https://docs.microsoft.com/dotnet/articles/standard/choosing-core-framework-server).
+## For more information
 
-If you’d like to connect with the ASP.NET Core development team’s progress and plans, tune in to the weekly [ASP.NET Community Standup](https://live.asp.net/).
+* [ASP.NET Core Tutorials](tutorials/index.md) to get started.
```
Comment (tdykstra) [inline aspnetcore/index.md]: In this new context remove "to get started", "for in-depth...".

File: aspnetcore/index.md [hunk 4]
```diff
@@ -16,66 +16,47 @@ uid: index
...
-
-## Next steps
-
-For getting-started tutorials, see [ASP.NET Core Tutorials](tutorials/index.md)
-
-For in-depth introduction to ASP.NET Core concepts and architecture, see [ASP.NET Core Fundamentals](fundamentals/index.md).
+ASP.NET Core is designed to integrate seamlessly with a variety of client-side frameworks, including [AngularJS](client-side/angular.md), <!-- KO is no longer featured --> [KnockoutJS](client-side/knockout.md) and [Bootstrap](client-side/bootstrap.md). See [Client-Side Development](client-side/index.md) for more details.
 
-An ASP.NET Core app can use the .NET Core or .NET Framework runtime. For more information, see [Choosing between .NET Core and .NET Framework](https://docs.microsoft.com/dotnet/articles/standard/choosing-core-framework-server).
+## For more information
```
Comment (tdykstra) [inline aspnetcore/index.md]: We never title this section "For more information" -- usually it's Next steps, though sometimes we use Additional resources. Here you could put "For more information, see the following resources" under a Next steps title.

File: aspnetcore/index.md [hunk 5]
```diff
@@ -16,66 +16,47 @@ uid: index
...
 
-If you’d like to connect with the ASP.NET Core development team’s progress and plans, tune in to the weekly [ASP.NET Community Standup](https://live.asp.net/).
+* [ASP.NET Core Tutorials](tutorials/index.md) to get started.
+* [ASP.NET Core Fundamentals](fundamentals/index.md) for in-depth information on concepts and architecture.
+* The weekly [ASP.NET Community Standup](https://live.asp.net/), which covers the team's progress and plans and features new blogs and third party software.
```
Comment (tdykstra) [inline aspnetcore/index.md]: remove "which"

File: aspnetcore/index.md [hunk 6]
```diff
@@ -16,66 +16,47 @@ uid: index
...
+ASP.NET Core is a new open-source and cross-platform framework for building modern cloud based internet connected applications, such as web apps, IoT apps and mobile backends. ASP.NET Core apps can run on [.NET Core](https://www.microsoft.com/net/core/platform) or on the full .NET Framework. It was architected to provide an optimized development framework for apps that are deployed to the cloud or run on-premises. It consists of modular components with minimal overhead, so you retain flexibility while constructing your solutions. You can develop and run your ASP.NET Core apps cross-platform on Windows, Mac and Linux. ASP.NET Core is open source at [GitHub](https://github.com/aspnet/home). An ASP.NET Core app can use the .NET Core or .NET Framework runtime. For more information, see [Choosing between .NET Core and .NET Framework](https://docs.microsoft.com/dotnet/articles/standard/choosing-core-framework-server).
```
Comment (Rick-Anderson) [inline aspnetcore/index.md]: Yes, I moved it from next steps. New paragraph good.

File: aspnetcore/index.md (hunk 2 above)
Comment (Rick-Anderson) [inline aspnetcore/index.md]: remove it. MIssed that one.

File: aspnetcore/index.md (hunk 6 above)
Comment (scottaddie) [inline aspnetcore/index.md]: In this 1st sentence, hyphenate "cloud based" and "internet based". Also, capitalize the "i" on "internet".

Should we be referring to ".NET Framework" as the "full .NET Framework"? Personally, I'd remove the word "full", now that .NET Standard 2.0 is introducing more APIs to be implemented in .NET Core 2.0.

File: aspnetcore/index.md [hunk 7]
```diff
@@ -16,66 +16,47 @@ uid: index
...
-* Ability to host on IIS or self-host in your own process
-
-* Built on [.NET Core](https://microsoft.com/net/core), which supports true side-by-side app versioning
-
-* Ships entirely as [NuGet](https://nuget.org)  packages
-
-* New tooling that simplifies modern web development
-
-* Build and run cross-platform ASP.NET apps on Windows, Mac and Linux
+ASP.NET Core provides the following foundational improvements:
```
Comment (scottaddie) [inline aspnetcore/index.md]: Looks like there are 2 spaces before the 2nd sentence. Remove 1 of them. Also recommend adding a comma after "architectural changes" and hyphenating "well factored".

File: aspnetcore/index.md (hunk 5 above)
Comment (scottaddie) [inline aspnetcore/index.md]: Hyphenate "third party".

File: aspnetcore/index.md (hunk 6 above)
Comment (tdykstra) [inline aspnetcore/index.md]: Yes, we're supposed to avoid "full" .NET Framework

**Pull request merged**